tests/test_enterprise/views/test_enterprise_proxy_login_view.py

```diff
@@ -63,7 +63,11 @@ def test_redirect_no_tpa(self):
         customer_without_tpa = EnterpriseCustomerFactory()
         url = self._get_url_with_params(enterprise_slug_override=customer_without_tpa.slug, use_next=False)
         response = self.client.get(url)
-        self.assertRedirects(response, LMS_LOGIN_URL, fetch_redirect_response=False)
+        query_params = QueryDict(mutable=True)
+        query_params['enterprise_customer'] = str(customer_without_tpa.uuid)
```

Comment:
would these query params also apply to the redirect for tpa below (line 72)?

Reply:
Not for the way it's currently implemented but once I address Brandon's comment they will 👍

Reply:
these are only for edX logistration to render the new sidebar, not needed for sso